<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default Suite" duration-ms="13812" started-at="2023-01-23T07:32:11 IST" finished-at="2023-01-23T07:32:25 IST">
    <groups>
    </groups>
    <test name="AppiumLearningProject" duration-ms="13812" started-at="2023-01-23T07:32:11 IST" finished-at="2023-01-23T07:32:25 IST">
      <class name="org.testngexample.AndroidBusinessValidationExample">
        <test-method status="PASS" signature="setup()[pri:0, instance:org.testngexample.AndroidBusinessValidationExample@5ae63ade]" name="setup" is-config="true" duration-ms="11862" started-at="2023-01-23T07:32:11 IST" finished-at="2023-01-23T07:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyAccessibilityOptionText()[pri:0, instance:org.testngexample.AndroidBusinessValidationExample@5ae63ade]" name="verifyAccessibilityOptionText" duration-ms="414" started-at="2023-01-23T07:32:23 IST" finished-at="2023-01-23T07:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAccessibilityOptionText -->
        <test-method status="FAIL" signature="verifyAnimationOptionText()[pri:0, instance:org.testngexample.AndroidBusinessValidationExample@5ae63ade]" name="verifyAnimationOptionText" duration-ms="510" started-at="2023-01-23T07:32:23 IST" finished-at="2023-01-23T07:32:24 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Animation] but found [Anmate]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Animation] but found [Anmate]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at org.testngexample.AndroidBusinessValidationExample.verifyAnimationOptionText(AndroidBusinessValidationExample.java:56)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAnimationOptionText -->
        <test-method status="FAIL" signature="verifyVeiwOptionText()[pri:0, instance:org.testngexample.AndroidBusinessValidationExample@5ae63ade]" name="verifyVeiwOptionText" duration-ms="327" started-at="2023-01-23T07:32:24 IST" finished-at="2023-01-23T07:32:24 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Views] but found [View]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Views] but found [View]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at org.testngexample.AndroidBusinessValidationExample.verifyVeiwOptionText(AndroidBusinessValidationExample.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyVeiwOptionText -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.testngexample.AndroidBusinessValidationExample@5ae63ade]" name="tearDown" is-config="true" duration-ms="650" started-at="2023-01-23T07:32:24 IST" finished-at="2023-01-23T07:32:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.testngexample.AndroidBusinessValidationExample -->
    </test> <!-- AppiumLearningProject -->
  </suite> <!-- Default Suite -->
</testng-results>
